main.tf

```diff
@@ -0,0 +1,18 @@
+provider "azurerm" {
+  features {}
+}
+
+# Create a resource group
+resource "azurerm_resource_group" "rg" {
+  name     = "my-resource-group"
```

Comment:
Can you change it to your name - say swapna01

Reply:
Hi...Changes made.

Reply:
approved!!
you can merge the code